apps/server/src/modules/h5p-editor/controller/h5p-editor.controller.ts

```diff
@@ -0,0 +1,45 @@
+import { Controller, Get } from '@nestjs/common';
+import { ApiTags } from '@nestjs/swagger';
+import { Authenticate } from '@src/modules/authentication/decorator/auth.decorator';
+
+// Dummy html response so we can test i-frame integration
+const dummyResponse = (title: string) => `
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>${title}</title>
+</head>
+<body>
+    <h1>${title}</h1>
+    <p>This response can be used for testing</p>
+</body>
+</html>
+`;
+
+@ApiTags('h5p-editor')
+@Authenticate('jwt')
+@Controller('h5p-editor')
+export class H5PEditorController {
+	@Get('/:contentId/play')
+	async getPlayer() {
+		// Dummy Response
+		return Promise.resolve(dummyResponse('H5P Player Dummy'));
+	}
+
+	@Get('/:contentId/edit')
+	async getEditor() {
```

Comment:
It is only a mock, but please execute the open api client. To avoid confusings in other teams. Please also check the settings that we use for the files-storage to get the isolated api client.

Reply:
Generated openapi client